tgui/packages/tgui/public/tgui.html

```diff
@@ -133,9 +133,12 @@
     <!-- Fatal error container -->
     <div id="FatalError" style="display: none">
       <div class="FatalError__logo">
-        ███╗ ██╗████████╗ ██████╗ ███████╗ ████╗ ██║╚══██╔══╝██╔═══██╗██╔════╝
-        ██╔██╗ ██║ ██║ ██║ ██║███████╗ ██║╚██╗██║ ██║ ██║ ██║╚════██║ ██║ ╚████║
-        ██║ ╚██████╔╝███████║ ╚═╝ ╚═══╝ ╚═╝ ╚═════╝ ╚══════╝
+        ███╗   ██╗████████╗ ██████╗ ███████╗
+        ████╗  ██║╚══██╔══╝██╔═══██╗██╔════╝
+        ██╔██╗ ██║   ██║   ██║   ██║███████╗
+        ██║╚██╗██║   ██║   ██║   ██║╚════██║
+        ██║ ╚████║   ██║   ╚██████╔╝███████║
+        ╚═╝  ╚═══╝   ╚═╝    ╚═════╝ ╚══════╝
       </div>
```

Comment:
Can you make sure this works normally after prettier is run?

Reply:
Ran prettier on this, the file remained unchanged. Good call!